frontend/packages/otr/src/components/clerkInterpreter/listing/ClerkInterpreterListingRow.tsx

```diff
@@ -24,24 +24,34 @@ export const ClerkInterpreterListingRow = ({
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
 
-  const { lastName, nickName, qualifications } = interpreter;
+  const { id, lastName, nickName, qualifications } = interpreter;
 
   const visibleQualifications =
     QualificationUtils.getQualificationsVisibleInClerkHomePage(qualifications);
 
-  const handleRowClick = (e: React.MouseEvent<HTMLTableRowElement>) => {
-    e?.stopPropagation();
-    navigate(
-      AppRoutes.ClerkInterpreterOverviewPage.replace(
-        /:interpreterId$/,
-        `${interpreter.id}`
-      )
-    );
+  const overviewUrl = AppRoutes.ClerkInterpreterOverviewPage.replace(
+    /:interpreterId$/,
+    `${id}`
+  );
+
+  const handleRowLeftClick = (e: React.MouseEvent<HTMLTableRowElement>) => {
+    e.stopPropagation();
```

Comment:
Yksi vaihtoehto olisi tsekata tässä onko meta-näppäin painettuna. 
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Reply:
Voisi varmaan Oton kanssa myös käydä läpi eri ratkaisumahdollisuuksia.

Reply:
Aivan joo.. tosin olisko tää lopulta paljonkaan parempi? Jos kuuntelisi klikkiä kuten tähänkin asti noin muuten.